### DISCUSSION TITLE
Is it normal that "Clean Build Folder" is required for new cases?

### DISCUSSION THREAD

**Discussion by notcome** [pointfreeco/swift-case-paths#112] [Q&A]
I am running into an issue where, when I add new cases to an existing Action type, I have to clean the build folder, or I will run into weird issues, including EXC_BAD_ACCESS and wrong case paths being matched. Is this normal?

**Comment by stephencelis** [comment]
I believe this is unfortunately the result of an incremental bug in the Swift compiler reported here: https://github.com/apple/swift/issues/60706

It might be worth leaving a comment that you encounter the issue regularly, as well. More visibility might lead to more prioritization.
